src/containers/darien/DarienProgram.jsx

```diff
@@ -0,0 +1,70 @@
+/*
+Darien Program
+--------------
+This container is the "main parent" that oversees/organises the rest of the
+components used by WildCam Darien Lab.
+
+Currently, it has two concerns:
+- Only allow non-logged-in users access to certain parts of the Lab.
+- Route Teachers and Students to their correct components.
+
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Switch, Route } from 'react-router-dom';
+import DarienHome from '../../components/darien/DarienHome';
+import DarienEducators from '../../components/darien/DarienEducators';
+import DarienMap from '../../components/darien/DarienMap';
+import Darien401 from '../../components/darien/Darien401';
+import Darien404 from '../../components/darien/Darien404';
+import GenericStatusPage from '../../components/common/GenericStatusPage';
+
+class DarienProgram extends React.Component {
```

Comment:
I think you can still use the higher order connect function with functional components, so I don't think this has to be a class, unless you have plans to add some lifecycle methods to handle something.